criu/include/kerndat.h

```diff
@@ -8,6 +8,9 @@
 #include "util-vdso.h"
 #include "hugetlb.h"
 
+#define RELEASE_SIZE 66
+#define NODENAME_SIZE 66
+
```

Comment:
Is this still needed?

Reply:
Nope, missed removing it. Done now, thanks.